client/nginx_test.go

```diff
@@ -224,9 +235,227 @@ func TestStreamDetermineUpdates(t *testing.T) {
 	}}
 
 	for _, test := range tests {
-		toAdd, toDelete := determineStreamUpdates(test.updated, test.nginx)
+		toAdd, toDelete, _ := determineStreamUpdates(test.updated, test.nginx)
 		if !reflect.DeepEqual(toAdd, test.expectedToAdd) || !reflect.DeepEqual(toDelete, test.expectedToDelete) {
 			t.Errorf("determiteUpdates(%v, %v) = (%v, %v)", test.updated, test.nginx, toAdd, toDelete)
 		}
 	}
 }
+
+func TestDetermineUpdatesWithUpdate(t *testing.T) {
```

Comment:
```suggestion
func TestDetermineUpdates(t *testing.T) {
```

Reply:
On second look, not sure my suggestion makes sense. The test calls `determineUpdates` which returns `toCreate`, `toDelete` and `toUpdate` but in this test we just test `toUpdate`.

Instead maybe you can take a look to see if these cases can be tested as additional cases in the already existing `TestDetermineUpdates`?

Same applies to: https://github.com/nginxinc/nginx-plus-go-client/pull/34#discussion_r340212786

Reply:
Yeah I am not sure why there are 2 methods, good catch, I'll merge them. It makes sense with the new feature 👍